### ISSUE TITLE
Quokka with Typescript and baseUrl

### ISSUE THREAD

**Issue by senser1** [wallabyjs/quokka#546]
### I'm trying to get Quokka (pro) running with TypeScript and a configured baseUrl property.

I'm using this starter project: https://github.com/wallabyjs/quokka-create-react-app-ts with Quokka. I only added the `baseUrl` property pointing to src.

```{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "esModuleInterop": true,
    "allowSyntheticDefaultImports": true,
    "strict": true,
    "baseUrl": "src",
    "forceConsistentCasingInFileNames": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "noEmit": true,
    "jsx": "preserve"
  },
  "include": ["src"]
}
```
If I try to import code from a folder in the root of `src` (for instance the test1.ts file inside 'src/helpers'), like this:

<snip from q.ts>

```
import { test } from "helpers/test1";

test1();
```

<end snip>

TypeScript is happy and all is fine. However, If I start Quokka on a file that uses an import like this, I get the error:

```
​​​​​Quokka 'q.ts' (node: v12.16.3, babel: v7.4.3, plugins: auto-detect:create-react-app, jsdom-quokka-plugin)​​​​
 
Cannot find module 'helpers/test1' 
Require stack: 
- ./src/q/q.ts 
  at ​​​Function.Module._resolveFilename​​​ ​internal/modules/cjs/loader.js:957​
  at ​​​Function.Module._load​​​ ​internal/modules/cjs/loader.js:840​
  at ​​​require​​​ ​internal/modules/cjs/helpers.js:77​
  at ​​​Object.<anonymous>​​​ ​src/q/q.ts:1​
  at ​​​Module._compile​​​ ​internal/modules/cjs/loader.js:1133​
```

**Comment by ArtemGovorov** [comment]
Thanks for reporting the issue. It is fixed in the latest VS Code extension v1.0.321.

**Comment by senser1** [comment]
Indeed fixed in the latest version. 👍
